control-plane/bfruntime.h

```diff
@@ -58,6 +58,7 @@ static inline constexpr P4Id getIdPrefix(P4Id id) {
 }
 
 static inline Util::JsonObject* findJsonTable(Util::JsonArray* tablesJson, cstring tblName) {
+    tblName = "pipe." + tblName;
```

Comment:
"pipe." is not a prefix, it comes from the name of the pipeline.

Reply:
Will work on this in a follow up PR to update the bfRT JSON output with fully qualified table names. Created an issue for the same.
https://github.com/p4lang/p4c/issues/2927